openedx/features/course_experience/views/course_home_messages.py

```diff
@@ -114,7 +114,7 @@ def register_course_home_messages(request, course, user_access, course_start_dat
             CourseHomeMessages.register_info_message(
                 request,
                 Text(_(
-                    "{add_reminder_open_tag}Don't forget to add a calendar reminder!{add_reminder_close_tag}."
+                    "{add_reminder_open_tag}Don't forget to add a calendar reminder!{add_reminder_close_tag}"
```

Comment:
@HarryRein Nit: Hopefully we'll be getting to this soon, but in general, it's not a good practice to leave useless code for a future implementation.  This really should be:
```
CourseHomeMessages.register_info_message(
    request,
    _("Don't forget to add a calendar reminder!")
)
```